src/app/modules/reports/components/time-entries-table/time-entries-table.component.spec.ts

```diff
@@ -76,6 +79,24 @@ describe('Reports Page', () => {
       expect(component.dtTrigger.next).toHaveBeenCalled();
     });
 
+    it('when the uri starts with http or https it should return true and open the url in a new tab', () => {
+      const uriExpected = 'http://customuri.com';
```

Comment:
In this test. I think it is not needed to cal the variable `uriExpected` as it is not being compared with other value, it may be just `url`, for readability.